tests/test_utils.py

```diff
@@ -0,0 +1,33 @@
+import pytest
```

Comment:
Sorry, I didn't get what you mean. Do you mean the test file is wrong, like not test_utils.py but test_requests.py?